### PR TITLE
Phpdoc*Fixer - unify rename fixers

### PR DESCRIPTION
**Pull request by SpacePossum**
closes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1476

### CODE REVIEW

File: src/Fixer/Phpdoc/PhpdocNoAliasTagFixer.php [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        foreach ($configuration as $from => $to) {
+            if (!is_string($from)) {
+                throw new InvalidFixerConfigurationException($this->getName(), 'Tag to replace must be a string.');
+            }
+
+            if (!is_string($to)) {
+                throw new InvalidFixerConfigurationException($this->getName(), sprintf('Tag to replace to from "%s" must be a string.', $from));
+            }
+
+            if (1 !== preg_match('#^[^\s]+$#', $to) || false !== strpos($to, '*/')) {
```
Comment (keradus) [inline src/Fixer/Phpdoc/PhpdocNoAliasTagFixer.php]: sorry, i don't get this line
especially part with phpdoc ending.

why not check is it matchin `[a-zA-Z0-9]` ? spaces are not allowed, but `foo*bar` is not allowed as well.

File: tests/Fixer/Phpdoc/PhpdocNoAliasTagFixerTest.php [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+            ),
+        );
+    }
+
+    /**
+     * @dataProvider provideVarToTypeCases
+     */
+    public function testVarToTypeFix($expected, $input = null)
+    {
+        $this->getFixer()->configure(array(
```
Comment (keradus) [inline tests/Fixer/Phpdoc/PhpdocNoAliasTagFixerTest.php]: same issue here... or maybe we could solve it another way than cloning ?

Comment (SpacePossum) [inline tests/Fixer/Phpdoc/PhpdocNoAliasTagFixerTest.php]: fixed by rebasing on top of you fix

**Comment by keradus** [comment]
UPGRADE file needs to be updated

File: README.rst [hunk 3]
```diff
@@ -469,6 +465,10 @@ Choose from the list of available rules:
 * **phpdoc_no_access** [@Symfony]
    | @access annotations should be omitted from phpdocs.
 
+* **phpdoc_no_alias_tag** [@Symfony]
```
Comment (GrahamCampbell) [inline README.rst]: not really a good name tbh, given you rename things to their aliases if you want

File: src/Fixer/Phpdoc/PhpdocNoAliasTagFixer.php [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+ */
+final class PhpdocNoAliasTagFixer extends AbstractFixer
+{
+    /**
+     * @var array<string, string>
+     */
+    private $configuration;
+
+    private static $defaultConfiguration = array(
+        'property-read' => 'property',
```
Comment (GrahamCampbell) [inline src/Fixer/Phpdoc/PhpdocNoAliasTagFixer.php]: these are not alias renames for example

Comment (SpacePossum) [inline src/Fixer/Phpdoc/PhpdocNoAliasTagFixer.php]: so what should the name be in your opinion?

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**